File: Python/A/1003A-polycarppockets.py
n = int(input())
numbers = list(map(int, input().split()))
count_map: dict[int, int] = {}
for num in numbers:
    count_map[num] = count_map.get(num, 0) + 1
max_count = max(count_map.values(), default=0)
print(max_count)

File: C++/A/1003A-polycarppockets.cpp
#include <bits/stdc++.h>
using namespace std;
int main()
{
    int n;
    cin >> n;
    vector<int> numbers;
    for (int i = 0; i < n; i++)
    {
        int x;
        cin >> x;
        numbers.push_back(x);
    }
    unordered_map<int, int> countMap;
    for (int num : numbers)
        countMap[num]++;

    int maxCount = countMap[numbers[0]];
    for (const auto &pair : countMap)
        if (pair.second > maxCount)
            maxCount = pair.second;

    cout << maxCount;
}
